Resources/Prototypes/Floof/Entities/Objects/Devices/pda.yml

```diff
@@ -17,3 +17,20 @@
       whitelist:
         tags:
         - Write
+
+- type: entity
+  parent: BasePDA
+  id: FreelancerPDA
+  name: Freelancer PDA
+  description: Refurbished standard PDA
+  components:
+  - type: HealthAnalyzer
+    scanDelay: 1
+    scanningEndSound:
+      path: "/Audio/Items/Medical/healthscanner.ogg"
```

Comment:
Replace
```
- type: entity
  parent: BasePDA
  id: FreelancerPDA
  name: Freelancer PDA
  description: Refurbished standard PDA
  components:
  - type: HealthAnalyzer
    scanDelay: 1
    scanningEndSound:
      path: "/Audio/Items/Medical/healthscanner.ogg"
```
with 
```
- type: entity
  parent: BaseMedicalPDA
  id: FreelancerPDA
  name: Freelancer PDA
  description: Refurbished standard PDA
  components:
```
This is to ensure that later changes to the function of medical PDAs are inherited. In fact, such changes have already happened since this PR was written.